### PR TITLE
Expose some universe vars to devtools

### PR DESCRIPTION
**Pull request by andrejak**
Shows exposed plugin states (for example route and local) in devtools state panel. At least partially closes https://github.com/prodo-ai/prodo/issues/118.

(in the below screenshot, the app has no internal state, just route)
Screenshot 2019-10-08 at 17 06 15

With empty app state, we could instead show this (ignore color change):
Screenshot 2019-10-08 at 17 07 12

### CODE REVIEW

**Comment by tdawes** [comment]
Plugins might need to define for themselves how their state should be rendered - e.g. for the firestore plugin, I wouldn't really want to see the `db_cache` (but perhaps would be fine if it was just called `db`)

**Comment by andrejak** [comment]
At the moment, the firestore plugin can customize it only to the extent of choosing which variables to expose (eg. call exposeUniverseVariables(["db", "db_cache"]) will expose both, whereas exposeUniverseVariables(["db"]) would keep db_cache hidden). (Neither has actually been added yet, I only added route and local so far)
Did you have in mind a custom rendering function that would filter data to a more granular level, or change how it's displayed?

File: packages/devtools-plugin/src/index.ts [hunk 1]
```diff
@@ -23,15 +23,12 @@ const postMessage = (message: DevMessage) => {
 const devToolsPlugin = <State>(): ProdoPlugin<
```
Comment (coffee-cup) [inline packages/devtools-plugin/src/index.ts]: I'm not sure this plugin needs to be parameritezed by the `State` anymore. I know that the State type is used for the `DevToolsUniverse`, but the plugin accesses more than just the state now.

Comment (andrejak) [reply packages/devtools-plugin/src/index.ts]: The problem is that if I remove all this, state stops being typed for projects that use the devtools. Happy to show you if it helps.

**Comment by andrejak** [comment]
@coffee-cup this should be ready for re-review :)

**Comment by coffee-cup** [comment]
Some minor suggestions
- Add a title above the normal state section
- If there is no state have a message saying so. _App state is empty_ possibily
- Add a minimal (grey and small) indication that the state is editable but the plugin state is not

```
State          (editable)
{
}

Plugin State
{
}
```

**Comment by andrejak** [comment]
Do these look allright to you?

1. Only app state, no plugin state
Screenshot 2019-10-10 at 12 04 30

2. Both app state and plugin state
Screenshot 2019-10-10 at 12 08 26

3. No app state (but plugin state)
Screenshot 2019-10-10 at 12 09 55

**Comment by coffee-cup** [comment]
I like them!

**Pull request merged**